src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.ts

```diff
@@ -22,22 +23,15 @@ import { ValidationConstants } from 'src/app/shared/constants/validation';
 export class CreateAchievementComponent implements OnInit, OnDestroy {
   readonly validationConstants = ValidationConstants;
   @Select(UserState.selectedWorkshop) workshop$: Observable<Workshop>;
+  @Select(UserState.approvedChildren) approvedChildren$: Observable<Child[]>;
```

Comment:
workshop$: Observable<Workshop>;, approvedChildren$: Observable<Child[]>; move it on the next row, so we will keep the same style code

Reply:
don't see you have fixed this comment